index.html

```diff
@@ -1,304 +1,327 @@
+              </div>
+          </div>
+        </div>
+      </section>
 
-            <ul class="accordion">
-                <li>
-                    <div class="accordion-container">
-                        <a class="skill--description-link" href="#"><span class="skill--description">Languages</span></a
->
-                        <ion-icon class="chevron-icons" name="chevron-down-sharp"></ion-icon>
-                    </div>
-                    <ul>
-                        <li class="skill">
-                            <img class="skill-image" src="images/skill_1.png" alt="" />
-                            <span class="skill-name">Javascript</span>
-                        </li>
-                        <li class="skill">
-                            <img class="skill-image" src="images/skill_2.png" alt="" />
-                            <span class="skill-name">HTML</span>
-                        </li>
-                        <li class="skill">
-                            <img class="skill-image" src="images/skill_3.png" alt="" />
-                            <span class="skill-name">CSS</span>
-                        </li>
-                    </ul>
-                </li>
-            </ul>
-
-            <div class="frameworks-container">
-                <a class="framework--title-link" href="#">
-                    <span class="framework--title">Frameworks</span></a>
-                <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
-            </div>
-            <hr />
-            <div class="skills-container">
-                <a class="skills--title-link" href="#">
-                    <span class="skills--title">Skills</span></a>
-                <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
-            </div>
-        </section>
-
-        <section class="contact" id="contact">
-            <div class="inner-div">
-                <h2>Contact me</h2>
-                <p id="inner-div">
-                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
-                </p>
-            </div>
-            <form action="https://formspree.io/html" id="form" class="form" method="post">
-                <ul class="inner-list">
+      <section class="sectorcontact">
+        <div class="contact_sub">
+            <div class="cont_act">
+                <div class="contact_items">
+                    <h2 class="contactsheader" id="contact">Contact me</h2>
+                    <p class="contact_paragraph">If you have an application you are interested in developing, a
+                    feature that you need built or a project that needs coding. I’d
+                    love to help with it</p>
+                </div>
+                <div class="form">
+                  <form action="https://formspree.io/f/noqsnoqs" method="POST" id="form" class="form">
```

Comment:
- [ ] the form submission link is not working kindly make sure you create an account on [formspree](https://formspree.io/) so you can get a form id that you will use instead of `/noqsnoqs`

![image](https://user-images.githubusercontent.com/89282221/187518257-5fc8c39f-dfce-4b51-b750-f72b8b2d3aa0.png)